ONSdigital/dp-import-api
# Beta/1.6.0

### What

Update api to use latest changes to go-ns for auditing, pass context as first variable in function calls and use context in logs. 

Also split api handlers into separate files for easier readability.

### How to review

Final check changes make sense and unit tests pass, also acceptance tests for publishing should also pass (in dp-api-tests branch cmd-develop).

To run acceptance tests, you will need to do the following:

1) Run dp-dataset-api on cmd-develop with `make acceptance-publishing`
2) Run dp-auth-api-stub on cmd-develop with `make debug`
3) Run dp-recipe-api on cmd-develop with `make debug`
4) Run dp-import-api on beta/1.6.0 with `make acceptance`
5) Lastly run the following command in root level of repo `dp-api-tests`: `cd publishing/importAPI/; go test ./...`

### Who can review

Anyone
